ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/cygwin/tasks/main.yml

```diff
@@ -106,20 +106,30 @@
       - 'C:\cygwin64\bin'
     scope: machine
     state: absent
+  when: (not cygwin_installed.stat.exists)
   tags: cygwin
 
 # NOTE: This construct is because an SQL Server entry is being added to
 # the path which has an erroneous quote in it which causes problems for SETX
+# See https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1401
+- name: Remove speech mark from the %PATH%
+  win_shell: 'setx /M PATH "%PATH:\"=%"'
```

Comment:
Running from a command line on my test machine
```
setx TESTPATH "%TESTPATH:\"=%"
```
did not work (the double quote I had in TESTPATH was not removed).
```
setx TESTPATH "%TESTPATH:"=%"
```
(no backslash) worked (the double quote I had in TESTPATH was removed).

Reply:
The `\` has to be in there to escape the `"` after it, otherwise Ansible interprets it as a literal `"` and fails. It effectively runs on the Windows machine as the second example you gave :-)